bundles/CoreBundle/src/DependencyInjection/PimcoreCoreExtension.php

```diff
@@ -320,4 +326,29 @@ private function configureAdapterFactories(ContainerBuilder $container, array $f
 
         $serviceLocator->setArgument(0, $arguments);
     }
+
+    /**
+     * Extract class definitions and prefixes if configuration has more than just a class definition
+     */
+    private function flattenConfigurationForTypeLoader(array $configuration) : array
+    {
+        $newConfiguration = [
+            'map' => [],
+        ];
+        if (isset($configuration['map'])) {
```

Comment:
Couldn't we just replace this with an array_map and array_filter to filter null values?